pkg/logql/unmarshal/unmarshal.go

```diff
@@ -51,9 +52,9 @@ func NewStream(s *loghttp.Stream) *logproto.Stream {
 }
 
 // NewEntry constructs a logproto.Entry from a Entry
-func NewEntry(e loghttp.Entry) logproto.Entry {
+func NewEntry(e lokimodel.Entry) logproto.Entry {
```

Comment:
I'd really like to see deduplication of code around logproto definitions, but I think this can be done in a second PR and I wouldn't mind doing it (mostly because it requires adjusting the proto definitions).

opened https://github.com/grafana/loki/issues/1691